osu.Game/Screens/OnlinePlay/Lounge/LoungeSubScreen.cs

```diff
@@ -297,29 +297,26 @@ private void onLeaving()
             popoverContainer.HidePopover();
         }
 
-        public void Join(Room room, string password, Action<Room> onSuccess = null, Action<string> onFailure = null)
+        public virtual void Join(Room room, string password, Action<Room> onSuccess = null, Action<string> onFailure = null) => Schedule(() =>
```

Comment:
ah i guess this `virtual` is for mocking. at the time i couldn't understand why it was there.